decidim-comments/app/frontend/comments/comment.component.jsx

```diff
@@ -0,0 +1,43 @@
+import { propType }    from 'graphql-anywhere';
+import gql             from 'graphql-tag';
+
+import commentFragment from './comment.fragment.graphql'
+
+/**
+ * @returns {ReactComponent} - A single comment component with the author info and the comment's body
+ */
+const Comment = ({ comment: { author, body, createdAt } }) => (
+  <article className="comment">
+    <div className="comment__header">
+      <div className="author-data">
+        <div className="author-data__main">
+          <div className="author author--inline">
+            <a className="author__avatar">
+              <img alt="avatar" src="../../assets/images/demo-avatar.jpg" />
```

Comment:
As commented offline I'm ok using something more lightweight like `react-user-avatar` and I was wrong about the "lorem ipsum" thing.